### PR TITLE
#1 Feature/dependency management

### PR DESCRIPTION
**Pull request by guidotheelen**
Python dependencies will be installed through requirements.txt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,15 +3,21 @@
 Color and Black-and-White Noto emoji fonts, and tools for working with them.
 
 ## Building NotoColorEmoji
+Building NotoColorEmoji requires Python 3, and uses virtual envorinments to manage dependencies. Use the following steps to get up and running.
```
Comment (RoelN) [inline README.md:6]: envorniment → environment

File: README.md [hunk 2]
```diff
@@ -3,15 +3,21 @@
...
+Create a virtual environment called `noto-emoji-env`:
 
-Then run make.  NotoColorEmoji is the default target.  It's suggested to use -j,
+	python3 -m venv noto-emoji-env
+
+Activate the virtual environment:
+
+	source noto-emoji-env/bin/activate
+
+Install the python requirements with:
```
Comment (RoelN) [inline README.md:16]: python → Python

File: README.md (hunk 1 above)
Comment (guidotheelen) [reply README.md:6]: ✍️

File: README.md (hunk 2 above)
Comment (guidotheelen) [reply README.md]: ✅

**Pull request merged**